bar-trender-front-end/src/views/MainPage.js

```diff
@@ -0,0 +1,69 @@
+// import React from "react";
+import React, { useEffect, useState } from 'react';
+
+// core components
+import MainNavbar from "../components/Navbars/MainNavbar.js";
+import MobileNavbar from "../components/Navbars/MobileNavbar.js";
+
+import MainHeader from "../components/Headers/MainHeader.js";
+import DefaultFooter from "../components/Footers/DefaultFooter.js";
+import List from "../components/List";
+import withListLoading from '../components/withListLoading';
+
+import DeviceIdentifier from 'react-device-identifier';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7123646)
```suggestion
import DeviceIdentifier from "react-device-identifier";
```